### DISCUSSION TITLE
Addition of Collapse/Expand Button to the Rounds Editor screen

### DISCUSSION THREAD

**Discussion by Alex3753** [ppy/osu#25700] [Ideas]
When tournaments go on, past rounds' mappools start taking up substantial space on the Rounds Editor screen.
I propose the addition of a button to collapse or expand the beatmaps of a certain round or all rounds altogether for a better overview.

Quick mock-ups i made:

![lazer-expand_collapse1](https://github.com/ppy/osu/assets/66335155/0eaa6347-2aba-4372-a555-fd06449a94bc)

![lazer-expand_collapse2](https://github.com/ppy/osu/assets/66335155/c64cc228-9a9d-4e19-8af0-954bd9b6065f)

![lazer-expand_collapse3](https://github.com/ppy/osu/assets/66335155/4b0b1480-dc68-4a7d-a781-72ac47ff2ae4)

The collapsed rounds could also show the number of beatmaps inside of it, in order to let Users know which rounds have beatmaps already added to them.

![lazer-expand_collapse4](https://github.com/ppy/osu/assets/66335155/11b8e767-2e48-4430-b18a-be3131880608)

**Comment by megahello** [comment]
If I remember correctly, teams editor also doesn't have any collapse buttons. This would be nice there as well.
